### PR TITLE
Add 'llvm-cov gcov' reads coverage data and displays the coverage information

### PR DESCRIPTION
**Pull request by pfagner**
…the coverage information

### CODE REVIEW

File: tests/coverage_lcov/test_lcov.sh
Comment (bitmouse) [inline tests/coverage_lcov/test_lcov.sh:28]: _-- NOTE: Drawing will be removed after approval script_

Note: this review will not get an approval with this comment and this note.

File: tests/coverage_lcov/test_lcov.sh
Comment (bitmouse) [inline tests/coverage_lcov/test_lcov.sh:32]: why do we need this?

File: tests/coverage_lcov/test_lcov.sh
Comment (bitmouse) [inline tests/coverage_lcov/test_lcov.sh:51]: NUMBER_THREADS=4

Please add "-j" somewhere else, or rename variable

File: tests/coverage_lcov/test_lcov.sh
Comment (bitmouse) [inline tests/coverage_lcov/test_lcov.sh:65]: what is the ${COMPILER}?
Do you create coverage report for each compiler? We don't want to do that. Pick one.

**Review by bitmouse — CHANGES_REQUESTED** [review]

File: tests/coverage_lcov/test_lcov.sh
Comment (pfagner) [reply tests/coverage_lcov/test_lcov.sh:28]: Okay. I will remove this drawing.

File: tests/coverage_lcov/test_lcov.sh
Comment (jacky309) [inline tests/coverage_lcov/test_lcov.sh:12]: The script should be runnable from anywhere, especially from folders which are outside of the source folder since we do not want to write build files into the source folder.

File: tests/coverage_lcov/test_lcov.sh
Comment (jacky309) [inline tests/coverage_lcov/test_lcov.sh:15]: DIR_SRC=$(cd $(dirname "$0")/..; pwd -P)

Comment (jacky309) [reply tests/coverage_lcov/test_lcov.sh:15]: then you can remove that outerdir stuff....

**Review by jacky309 — CHANGES_REQUESTED** [review]

File: tests/coverage_lcov/test_lcov.sh
Comment (pfagner) [reply tests/coverage_lcov/test_lcov.sh:15]: We have already discussed the "related path .." and how to deduce the source path from the dirname.
If the this variant you proposed suits you, then this is not a problem.
Thank you.

Comment (jacky309) [reply tests/coverage_lcov/test_lcov.sh:15]: It is simple and it does work, so why would we need anything else ? Why did you change that in the first place ?

File: tests/coverage_lcov/read.me
Comment (jacky309) [inline tests/coverage_lcov/read.me:1]: Please do not insert lcov doc here. Simply provide a link to it.

File: tests/coverage_lcov/test_lcov.sh
Comment (pfagner) [reply tests/coverage_lcov/test_lcov.sh:15]: Q: Why did you change that in the first place?
A: I did to conclusion from your short sentences that is required is more "smart" solution for non trivial decision with traverse of folders.
J'ai pris vos paroles trop au sérieux. Vous vouliez le plus simple, je voulais le meilleur. Ne vous fâchez pas, tout ira bien de toute façon.

File: tests/coverage_lcov/read.me
Comment (pfagner) [reply tests/coverage_lcov/read.me:1]: Okay.

**Comment by pfagner** [comment]
This commit is closed due to no need to use two LLVM-COV + GCOV tools. Instead of it will be use LLVM-COV only:
https://github.com/Pelagicore/facelift/pull/302
All notes will be implement in related tool/commit.

**Pull request closed without merging**
